mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: {"AS", "F", "IS", "M_AS", "M_IS", "W"}
+
+    Returns
+    -------
+    intervals : pandas.DataFrame
+        All the data from the feature directory into one data frame
+        columns are strain, mouse, day, start, stop
+    """
+    # get all file names
```

Comment:
We should add a line of exception error handling to ensure that only **one** of the `{"AS", "F", "IS", "M_AS", "M_IS", "W"}` is entered in the input and exit gracefully with a message like _ "input value must be one of {"AS", "F", "IS", "M_AS", "M_IS", "W"}"_.

Here is [a guide](http://hplgit.github.io/primer.html/doc/pub/input/._input-bootstrap007.html) to doing this.